core/title_loader.py

```diff
@@ -58,6 +58,7 @@ def load_record(record):
 
         map_xml(load_record, urllib.request.urlopen(location))
 
+    @transaction.atomic
```

Comment:
looks like this is already being used in load_copyright and load_copyright_map, but I was curious if it's actually working the way we think it should be?  The docs say `Set ATOMIC_REQUESTS to True in the configuration of each database for which you want to enable this behavior.` and I don't see any config about ATOMIC_REQUESTS in our django settings files?  Should those be added somewhere?

https://docs.djangoproject.com/en/2.2/ref/settings/#std:setting-DATABASE-ATOMIC_REQUESTS

Reply:
That's a different thing.  The `@transaction.atomic` decorator means "this function is wrapped in a database transaction."  The setting to which you refer says "all view functions are wrapped in a database transaction."  The setting is a catch-all that we definitely do *not* want: https://docs.djangoproject.com/en/2.2/topics/db/transactions/#tying-transactions-to-http-requests

I tested that decorator, and it seemed to work as expected: if the function throws an exception, nothing is saved to the database, rather than the current situation where a partial record is saved.

Reply:
Ahhhhh okay cool, yeah I misunderstood